frontend/src/features/mapSideBar/containers/MotiInventoryContainer.tsx

```diff
@@ -193,7 +193,7 @@ export const MotiInventoryContainer: React.FunctionComponent = () => {
         toast.error(
           'Property search failed. Please check your search criteria and try again. If this error persists, contact the Help Desk.',
         );
-        console.debug(error);
+        // console.debug(error);
```

Comment:
If this was as a result of a warning, just delete it. I don't see a reason to add it back.